modules/fiftyone_degrees/device_detection/fiftyone_device_types.go

```diff
@@ -0,0 +1,78 @@
+package device_detection
+
+import (
+	"github.com/prebid/openrtb/v20/adcom1"
+)
+
+type deviceTypeMap = map[DeviceType]adcom1.DeviceType
+
+var mobileOrTabletDeviceTypes = []DeviceType{
+	DeviceTypeMobile,
+	DeviceTypeSmartPhone,
+}
+
+var personalComputerDeviceTypes = []DeviceType{
+	DeviceTypeDesktop,
+	DeviceTypeEReader,
+	DeviceTypeVehicleDisplay,
+}
+
+var tvDeviceTypes = []DeviceType{
+	DeviceTypeTv,
+}
+
+var phoneDeviceTypes = []DeviceType{
+	DeviceTypePhone,
+}
+
+var tabletDeviceTypes = []DeviceType{
+	DeviceTypeTablet,
+}
+
+var connectedDeviceTypes = []DeviceType{
+	DeviceTypeConsole,
+	DeviceTypeIoT,
+	DeviceTypeRouter,
+	DeviceTypeSmallScreen,
+	DeviceTypeSmartSpeaker,
+	DeviceTypeSmartWatch,
+}
+
+var setTopBoxDeviceTypes = []DeviceType{
+	DeviceTypeMediaHub,
+	DeviceTypeConsole,
+}
+
+var oohDeviceTypes = []DeviceType{
+	DeviceTypeKiosk,
+}
+
+func applyCollection(items []DeviceType, value adcom1.DeviceType, mappedCollection deviceTypeMap) {
+	for _, item := range items {
+		mappedCollection[item] = value
+	}
+}
+
+var deviceTypeMapCollection = deviceTypeMap{}
+
+// fiftyOneDtToRTB converts a 51Degrees device type to an OpenRTB device type.
+// If the device type is not recognized, it defaults to PC.
```

Comment:
Can you move this comment down so it is right above the `fiftyOneDtToRTB` function?